prisma/schema.prisma

```diff
@@ -3,15 +3,17 @@
 
 generator client {
     provider = "prisma-client-js"
+    previewFeatures = ["referentialIntegrity"]
```

Comment:
What is this for? Not saying it's wrong just curious what it does

Reply:
Per the docs: "Referential integrity is a property of a data set that states that all its references are valid. Referential integrity requires that if one record references another, then the referenced record must exist." To my understanding, it's there to enforce references on both related tables and prohibits you from deleting a table or row that has references pointed to it.

Reference: https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-integrity